src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -121,11 +137,12 @@
      * @param { DOM } element
      */
     var isBackground = function isBackground(element) {
-      return $(element).hasClass('BookReader')
+      var isBackgroundClick = $(element).hasClass('BookReader')
         || $(element).hasClass('BRcontainer') /* main black theatre */
         || $(element).hasClass('BRemptypage') /* empty page placeholder */
         || $(element).hasClass('BRpageview') /* empty page placeholder, 1up */
         || $(element).hasClass('BRtwopageview'); /* empty page placeholder, 2up */
+        return isBackgroundClick;
```

Comment:
Could you outdent this line a smidge? I lost it in `||` clauses.

Reply:
done